editor/store/test/reducer.js

```diff
@@ -1211,21 +1210,19 @@ describe( 'state', () => {
 			} );
 
 			expect( state ).toEqual( {
-				recentInserts: [
-					{ name: 'core-embed/twitter' },
-				],
 				insertUsage: {
 					'core-embed/twitter': {
+						time: expect.any( Number ),
```

Comment:
To the [above point](https://github.com/WordPress/gutenberg/pull/5342/files#r172350724), we shouldn't have any need for `expect.any` in this file, given that a reducer should be deterministic.